cmd/suffuse/main.go

```diff
@@ -1,11 +1,18 @@
 package main
 
-import . "github.com/suffuse/go-suffuse/suffuse"
+import (
+  . "github.com/suffuse/go-suffuse/suffuse"
+  "os"
+)
 
 func main() {
-  opts := ParseSfsOpts()
+  opts, opts_err := ParseSfsOpts(os.Args)
```

Comment:
I think you can just call it err here, and further down where you do `mfs, err := ...` it will allow it and re-use err while creating the slot for mfs. Possible I'm wrong about that but if it works I think it's better.